doc/md/privacy.md

```diff
@@ -452,12 +456,11 @@ We finish our example with another privacy-rule named `DenyMismatchedTenants` on
 The `DenyMismatchedTenants` rule rejects the group creation if the associated users don't belong to
 the same tenant as the group.
 
-```go
-// DenyMismatchedTenants is a rule runs only on create operations, and returns a deny decision
-// if the operation tries to add users to groups that are not in the same tenant.
+```go title="examples/privacytenant/rule/rule.go"
+// DenyMismatchedTenants is a rule that returns a deny decision if the operation
+// involves users that don't belong to the same tenant as the group or users from more than one tenant.
```

Comment:
Please mention that this rule is executed only on `Create` operations. See this line:

```go
privacy.OnMutationOperation(rule, ent.OpCreate)
```

Reply:
I made sure it is mentioned in the docs + in the code.